2023-07-11 12:08:45,860 INFO    StreamThr :106290 [internal.py:wandb_internal():86] W&B internal server running at pid: 106290, started at: 2023-07-11 12:08:45.859869
2023-07-11 12:08:45,864 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status
2023-07-11 12:08:45,866 INFO    WriterThread:106290 [datastore.py:open_for_write():85] open: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/run-2penfu2z.wandb
2023-07-11 12:08:45,867 DEBUG   SenderThread:106290 [sender.py:send():369] send: header
2023-07-11 12:08:45,867 DEBUG   SenderThread:106290 [sender.py:send():369] send: run
2023-07-11 12:08:46,851 INFO    SenderThread:106290 [dir_watcher.py:__init__():211] watching files in: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files
2023-07-11 12:08:46,852 INFO    SenderThread:106290 [sender.py:_start_run_threads():1100] run started: 2penfu2z with start time 1689048525.859204
2023-07-11 12:08:46,852 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: summary_record
2023-07-11 12:08:46,852 INFO    SenderThread:106290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-11 12:08:46,862 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: check_version
2023-07-11 12:08:46,862 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: check_version
2023-07-11 12:08:47,053 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: run_start
2023-07-11 12:08:47,057 DEBUG   HandlerThread:106290 [system_info.py:__init__():31] System info init
2023-07-11 12:08:47,057 DEBUG   HandlerThread:106290 [system_info.py:__init__():46] System info init done
2023-07-11 12:08:47,057 INFO    HandlerThread:106290 [system_monitor.py:start():181] Starting system monitor
2023-07-11 12:08:47,058 INFO    HandlerThread:106290 [system_monitor.py:probe():201] Collecting system info
2023-07-11 12:08:47,058 INFO    SystemMonitor:106290 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-07-11 12:08:47,060 INFO    SystemMonitor:106290 [interfaces.py:start():190] Started cpu monitoring
2023-07-11 12:08:47,062 INFO    SystemMonitor:106290 [interfaces.py:start():190] Started disk monitoring
2023-07-11 12:08:47,062 INFO    SystemMonitor:106290 [interfaces.py:start():190] Started gpu monitoring
2023-07-11 12:08:47,064 INFO    SystemMonitor:106290 [interfaces.py:start():190] Started memory monitoring
2023-07-11 12:08:47,064 INFO    SystemMonitor:106290 [interfaces.py:start():190] Started network monitoring
2023-07-11 12:08:47,074 DEBUG   HandlerThread:106290 [system_info.py:probe():195] Probing system
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [gitlib.py:_init_repo():53] git repository is invalid
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [system_info.py:probe():240] Probing system done
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [system_monitor.py:probe():210] {'os': 'Linux-5.19.0-46-generic-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-07-11T04:08:47.075069', 'startedAt': '2023-07-11T04:08:45.850776', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/zhuhe/HPE-with-Diffusion/main.py', 'codePath': 'main.py', 'host': '415-2', 'username': 'zhuhe', 'executable': '/home/zhuhe/.conda/envs/diffhpe/bin/python', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 3756.1938333333333, 'min': 800.0, 'max': 4783.333333333333}, 'cpu_freq_per_core': [{'current': 5097.197, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.221, 'min': 800.0, 'max': 5100.0}, {'current': 5100.25, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5099.606, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5200.0}, {'current': 5089.17, 'min': 800.0, 'max': 5200.0}, {'current': 5071.657, 'min': 800.0, 'max': 5200.0}, {'current': 5100.0, 'min': 800.0, 'max': 5200.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 3699.963, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}], 'disk': {'total': 93.31179428100586, 'used': 0.019802093505859375}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 31.185558319091797}}
2023-07-11 12:08:47,079 INFO    HandlerThread:106290 [system_monitor.py:probe():211] Finished collecting system info
2023-07-11 12:08:47,079 INFO    HandlerThread:106290 [system_monitor.py:probe():214] Publishing system info
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [system_info.py:_save_pip():67] Saving pip packages done
2023-07-11 12:08:47,079 DEBUG   HandlerThread:106290 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-07-11 12:08:47,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/conda-environment.yaml
2023-07-11 12:08:47,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/requirements.txt
2023-07-11 12:08:47,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/wandb-summary.json
2023-07-11 12:08:47,962 DEBUG   HandlerThread:106290 [system_info.py:_save_conda():86] Saving conda packages done
2023-07-11 12:08:47,964 INFO    HandlerThread:106290 [system_monitor.py:probe():216] Finished publishing system info
2023-07-11 12:08:47,966 DEBUG   SenderThread:106290 [sender.py:send():369] send: files
2023-07-11 12:08:47,966 INFO    SenderThread:106290 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-07-11 12:08:47,973 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: stop_status
2023-07-11 12:08:47,974 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: stop_status
2023-07-11 12:08:48,482 ERROR   wandb-upload_0:106290 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/2penfu2z/wandb-metadata.json?Expires=1689134928&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dTY%2BqKsJRCAy4rWmoNQINeOrZtzXf2XhLUR86Fy91taZ3PndOluP%2FjPtWX3K8ObFX5IYB1qxYyedh3rdh8yj3Y1F4zyDzZRrtBAMmME1wH8iXX%2F6RrzjUopYT%2BzWEMfeC%2FcWczr2qLUZd1iaJto2dAtt3J9oRj0Gz6TTW6Gp4DmNQkta%2BFX1R3XufcaRdmZr9eN%2FOMhFt870lZgB5EelHkPHbuBPTd9OrKtBKQD%2FOswSQZDSghSSWP9ls16lvZvdo3pljaLR93oZEw7YwQ2pit4XE4LKwVLsvLDDYg8lixMtq%2BwpRjJAHH57plyNOm3ZLwvVuSQwXPmuQtuP5wclaA%3D%3D: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/2penfu2z/wandb-metadata.json?Expires=1689134928&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dTY%2BqKsJRCAy4rWmoNQINeOrZtzXf2XhLUR86Fy91taZ3PndOluP%2FjPtWX3K8ObFX5IYB1qxYyedh3rdh8yj3Y1F4zyDzZRrtBAMmME1wH8iXX%2F6RrzjUopYT%2BzWEMfeC%2FcWczr2qLUZd1iaJto2dAtt3J9oRj0Gz6TTW6Gp4DmNQkta%2BFX1R3XufcaRdmZr9eN%2FOMhFt870lZgB5EelHkPHbuBPTd9OrKtBKQD%2FOswSQZDSghSSWP9ls16lvZvdo3pljaLR93oZEw7YwQ2pit4XE4LKwVLsvLDDYg8lixMtq%2BwpRjJAHH57plyNOm3ZLwvVuSQwXPmuQtuP5wclaA%3D%3D
2023-07-11 12:08:48,482 ERROR   wandb-upload_0:106290 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3618'}
2023-07-11 12:08:48,482 ERROR   wandb-upload_0:106290 [internal_api.py:upload_file():2196] upload_file response body: b'<!DOCTYPE html>\n<html lang=en>\n  <meta charset=utf-8>\n  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">\n  <title>Error 404 (Not Found)!!1</title>\n  <style>\n    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}\n  </style>\n  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>\n  <p><b>404.</b> <ins>That\xe2\x80\x99s an error.</ins>\n  <p>The requested URL <code>/wandb-production.appspot.com/ada4321/DiffHPE/2penfu2z/wandb-metadata.json</code> was not found on this server.  <ins>That\xe2\x80\x99s all we know.</ins>\n'
2023-07-11 12:08:48,612 DEBUG   SenderThread:106290 [sender.py:send():369] send: telemetry
2023-07-11 12:08:48,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/conda-environment.yaml
2023-07-11 12:08:48,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:08:48,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/wandb-metadata.json
2023-07-11 12:08:49,015 ERROR   wandb-upload_0:106290 [upload_job.py:push():136] Failed to upload file: /tmp/tmp855sv580wandb/cajrk0tb-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/2penfu2z/wandb-metadata.json?Expires=1689134928&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dTY%2BqKsJRCAy4rWmoNQINeOrZtzXf2XhLUR86Fy91taZ3PndOluP%2FjPtWX3K8ObFX5IYB1qxYyedh3rdh8yj3Y1F4zyDzZRrtBAMmME1wH8iXX%2F6RrzjUopYT%2BzWEMfeC%2FcWczr2qLUZd1iaJto2dAtt3J9oRj0Gz6TTW6Gp4DmNQkta%2BFX1R3XufcaRdmZr9eN%2FOMhFt870lZgB5EelHkPHbuBPTd9OrKtBKQD%2FOswSQZDSghSSWP9ls16lvZvdo3pljaLR93oZEw7YwQ2pit4XE4LKwVLsvLDDYg8lixMtq%2BwpRjJAHH57plyNOm3ZLwvVuSQwXPmuQtuP5wclaA%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-11 12:08:49,224 ERROR   Thread-16 :106290 [step_upload.py:_handle_event():200] Failed to upload file: /tmp/tmp855sv580wandb/cajrk0tb-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/2penfu2z/wandb-metadata.json?Expires=1689134928&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=dTY%2BqKsJRCAy4rWmoNQINeOrZtzXf2XhLUR86Fy91taZ3PndOluP%2FjPtWX3K8ObFX5IYB1qxYyedh3rdh8yj3Y1F4zyDzZRrtBAMmME1wH8iXX%2F6RrzjUopYT%2BzWEMfeC%2FcWczr2qLUZd1iaJto2dAtt3J9oRj0Gz6TTW6Gp4DmNQkta%2BFX1R3XufcaRdmZr9eN%2FOMhFt870lZgB5EelHkPHbuBPTd9OrKtBKQD%2FOswSQZDSghSSWP9ls16lvZvdo3pljaLR93oZEw7YwQ2pit4XE4LKwVLsvLDDYg8lixMtq%2BwpRjJAHH57plyNOm3ZLwvVuSQwXPmuQtuP5wclaA%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-11 12:08:50,855 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:08:51,407 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:08:52,856 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:08:56,630 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:08:59,650 DEBUG   SenderThread:106290 [sender.py:send():369] send: telemetry
2023-07-11 12:09:00,859 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:09:01,718 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:09:02,971 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: stop_status
2023-07-11 12:09:02,972 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: stop_status
2023-07-11 12:09:06,861 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:09:07,111 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:09:08,862 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:09:12,595 DEBUG   SenderThread:106290 [sender.py:send():369] send: exit
2023-07-11 12:09:12,596 INFO    SenderThread:106290 [sender.py:send_exit():574] handling exit code: 0
2023-07-11 12:09:12,596 INFO    SenderThread:106290 [sender.py:send_exit():576] handling runtime: 25
2023-07-11 12:09:12,597 INFO    SenderThread:106290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-11 12:09:12,598 INFO    SenderThread:106290 [sender.py:send_exit():582] send defer
2023-07-11 12:09:12,598 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,598 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 0
2023-07-11 12:09:12,598 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:09:12,599 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,599 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 0
2023-07-11 12:09:12,599 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 1
2023-07-11 12:09:12,599 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,599 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 1
2023-07-11 12:09:12,599 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,599 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 1
2023-07-11 12:09:12,599 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 2
2023-07-11 12:09:12,599 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,599 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 2
2023-07-11 12:09:12,599 INFO    HandlerThread:106290 [system_monitor.py:finish():190] Stopping system monitor
2023-07-11 12:09:12,599 DEBUG   SystemMonitor:106290 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-07-11 12:09:12,599 DEBUG   SystemMonitor:106290 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2023-07-11 12:09:12,600 DEBUG   SystemMonitor:106290 [system_monitor.py:_start():170] Publishing last batch of metrics
2023-07-11 12:09:12,600 INFO    HandlerThread:106290 [interfaces.py:finish():202] Joined cpu monitor
2023-07-11 12:09:12,601 INFO    HandlerThread:106290 [interfaces.py:finish():202] Joined disk monitor
2023-07-11 12:09:12,663 INFO    HandlerThread:106290 [interfaces.py:finish():202] Joined gpu monitor
2023-07-11 12:09:12,663 INFO    HandlerThread:106290 [interfaces.py:finish():202] Joined memory monitor
2023-07-11 12:09:12,663 INFO    HandlerThread:106290 [interfaces.py:finish():202] Joined network monitor
2023-07-11 12:09:12,664 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,664 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 2
2023-07-11 12:09:12,664 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 3
2023-07-11 12:09:12,664 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,664 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 3
2023-07-11 12:09:12,665 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,665 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 3
2023-07-11 12:09:12,665 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 4
2023-07-11 12:09:12,665 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,665 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 4
2023-07-11 12:09:12,665 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,665 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 4
2023-07-11 12:09:12,665 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 5
2023-07-11 12:09:12,666 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,666 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 5
2023-07-11 12:09:12,666 DEBUG   SenderThread:106290 [sender.py:send():369] send: summary
2023-07-11 12:09:12,667 INFO    SenderThread:106290 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-11 12:09:12,667 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,667 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 5
2023-07-11 12:09:12,667 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 6
2023-07-11 12:09:12,667 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,667 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 6
2023-07-11 12:09:12,668 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,668 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 6
2023-07-11 12:09:12,673 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: status_report
2023-07-11 12:09:12,863 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:09:12,863 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/wandb-summary.json
2023-07-11 12:09:12,970 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 7
2023-07-11 12:09:12,970 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:12,970 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 7
2023-07-11 12:09:12,970 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:12,970 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 7
2023-07-11 12:09:13,595 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: poll_exit
2023-07-11 12:09:13,863 INFO    Thread-17 :106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/config.yaml
2023-07-11 12:09:16,734 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 8
2023-07-11 12:09:16,734 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: poll_exit
2023-07-11 12:09:16,734 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:16,735 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 8
2023-07-11 12:09:16,735 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:16,735 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 8
2023-07-11 12:09:16,735 INFO    SenderThread:106290 [job_builder.py:build():232] Attempting to build job artifact
2023-07-11 12:09:16,736 INFO    SenderThread:106290 [job_builder.py:build():266] no source found
2023-07-11 12:09:16,736 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 9
2023-07-11 12:09:16,736 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:16,736 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 9
2023-07-11 12:09:16,736 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:16,737 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 9
2023-07-11 12:09:16,737 INFO    SenderThread:106290 [dir_watcher.py:finish():359] shutting down directory watcher
2023-07-11 12:09:16,865 INFO    SenderThread:106290 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log
2023-07-11 12:09:16,865 INFO    SenderThread:106290 [dir_watcher.py:finish():389] scan: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files
2023-07-11 12:09:16,866 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/wandb-metadata.json wandb-metadata.json
2023-07-11 12:09:16,866 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/config.yaml config.yaml
2023-07-11 12:09:16,866 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/requirements.txt requirements.txt
2023-07-11 12:09:16,866 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/wandb-summary.json wandb-summary.json
2023-07-11 12:09:16,867 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/conda-environment.yaml conda-environment.yaml
2023-07-11 12:09:16,867 INFO    SenderThread:106290 [dir_watcher.py:finish():403] scan save: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230711_120845-2penfu2z/files/output.log output.log
2023-07-11 12:09:16,868 INFO    SenderThread:106290 [sender.py:transition_state():602] send defer: 10
2023-07-11 12:09:16,875 DEBUG   HandlerThread:106290 [handler.py:handle_request():144] handle_request: defer
2023-07-11 12:09:16,875 INFO    HandlerThread:106290 [handler.py:handle_request_defer():170] handle defer: 10
2023-07-11 12:09:16,875 DEBUG   SenderThread:106290 [sender.py:send_request():396] send_request: defer
2023-07-11 12:09:16,875 INFO    SenderThread:106290 [sender.py:send_request_defer():598] handle sender defer: 10
2023-07-11 12:09:16,875 INFO    SenderThread:106290 [file_pusher.py:finish():159] shutting down file pusher
